2-alcance/README.md

```diff
@@ -1,3 +1,112 @@
 # Plano de alcance
 
-## Referencias
\ No newline at end of file
+## Especificaciones funcionales
+
+								Musikamania
+							Especificación Funcional
+
+								Pepito García
+						Última actualización: 4 de noviembre, 2018
+
+### Generalidades
+
+<i>Musikamania.com</i> es un servicio, vía web, que cuenta con contenidos musicales como reportajes, entrevistas, noticias, incluso listas musicales de distintos géneros. 
+Esta especificación no está terminada. Todo el texto del interfaz deber ser revisado varias veces antes de darse por acabada. El aspecto final se desarrollará a lo largo del tiempo, en colaboración de los diseñadores gráficos y del estudio de las reacciones de los usuarios.
+
+### Escenarios
+
+Escenario 1: Elena es una chica muy tranquila, y cuando llega a casa tras pasar todo el día dando clase a los niños, le gusta tumbarse o darse un baño relajante con música tranquila. No le gustan las páginas webs que suele utilizar, porque aparece demasiada publicidad o no encuentra demasiada música de su gusto.
```

Comment:
Aquí (y en los escenarios posteriores), más que repetir la información que ya habéis presentado en el apartado anterior (al definir las personas), deberías especificar en qué contexto esta persona podría hacer uso de un producto como el vuestro. Por ejemplo:

> Elena llega a casa tarde después de un agotador día de trabajo y decide tomarse un baño antes de ponerse con la cena. Entra en Musikamania.com usando su teléfono móvil, selecciona la lista más popular de música relajante y le da al play. Conecta el teléfono al hilo musical de su casa, y se mete en el baño.

¿Qué implicaciones tiene un escenario como éste? Por ejemplo, Elena no está delante de la pantalla para interactuar con el sitio web, ¿seguirá sonando la música cuando acabe la primera canción? (por poner un ejemplo).